.gitmodules

```diff
@@ -66,10 +66,6 @@
     ignore = dirty
     path = third_party/python-peachpy
     url = https://github.com/malfet/PeachPy.git
-[submodule "third_party/python-six"]
```

Comment:
I actually don't think this is our dependency, but a dependency of one of the other third_party libs so it make break it. Let's see if it does though.

Reply:
It's a dependency of `cmake/External/nnpack.cmake`:

https://github.com/pytorch/pytorch/blob/a8fdfb4ba8a804c67d744a763fd9fa1f72d28590/cmake/External/nnpack.cmake#L48

re-added it.

Reply:
Hmm, we can always submit a PR to nnpack to fix the dependency on their end